File: aurora-common/src/main/java/xyz/xcye/common/vo/RoleVO.java
package xyz.xcye.common.vo;

import lombok.Data;
import xyz.xcye.common.constant.FieldLengthConstant;
import xyz.xcye.common.valid.validator.ValidateString;

@Data
public class RoleVO {
    /**
     * 唯一uid，不需要雪花生成
     */
    private Integer uid;

    /**
     * 角色或者权限
     */
    private String role;

    /**
     * 创建时间
     */
    private String createTime;

    /**
     * 最后更新时间
     */
    private String updateTime;
}

File: src/theme/vo/admin/RoleVo.ts
interface RoleVo {

  /**
   * 唯一uid，自增
   */
  uid?: string | null;

  /**
   * 角色的名称，不用添加ROLE_
   */
  name?: string | null;

  /**
   * 
   */
  createTime?: string | null;

  /**
   * 
   */
  updateTime?: string | null;

  /**
   * 用户的状态 1：已禁用 0：未禁用
   */
  status?: boolean | null;

}

export { RoleVo }